### PR TITLE
misc: various STACKIT related improvements

### PR DESCRIPTION
**Pull request by malt3**
### Context

Improvements for STACKIT. Small changes are collected here.

### Proposed change(s)

- cli: add STACKIT to constellation config instance-types
- misc: skip message about community license with marketplace image
- cli: correct measurements in `config generate stackit`

### Checklist

- [x] Run the E2E tests that are relevant to this PR's changes
- [x] Update [docs](https://github.com/edgelesssys/constellation/tree/main/docs)
- [x] Add labels (e.g., for changelog category)
- [x] Is PR title adequate for changelog?
- [x] Link to Milestone

### CODE REVIEW

**Comment by malt3** [comment]
CC @thomasten for 61bb09466a65a99c28610e6aa5016df6da7a00bd

File: cli/internal/cmd/license_enterprise.go [hunk 1]
```diff
@@ -22,18 +22,22 @@ import (
...
-func (a *applyCmd) checkLicenseFile(cmd *cobra.Command, csp cloudprovider.Provider) {
+func (a *applyCmd) checkLicenseFile(cmd *cobra.Command, csp cloudprovider.Provider, useMarketplaceImage bool) {
 	var licenseID string
 	a.log.Debug("Running license check")
 
 	readBytes, err := a.fileHandler.Read(constants.LicenseFilename)
-	if errors.Is(err, fs.ErrNotExist) {
+	switch {
+	case useMarketplaceImage:
+		cmd.Printf("Using marketplace image billing.\n")
```
Comment (msanft) [inline cli/internal/cmd/license_enterprise.go:32]:
```suggestion
		cmd.Println("Using marketplace image billing.")
```

File: internal/config/config.go [hunk 2]
```diff
@@ -720,7 +720,8 @@ func (c *Config) DeployYawolLoadBalancer() bool {
 func (c *Config) UseMarketplaceImage() bool {
 	return (c.Provider.Azure != nil && c.Provider.Azure.UseMarketplaceImage != nil && *c.Provider.Azure.UseMarketplaceImage) ||
 		(c.Provider.GCP != nil && c.Provider.GCP.UseMarketplaceImage != nil && *c.Provider.GCP.UseMarketplaceImage) ||
-		(c.Provider.AWS != nil && c.Provider.AWS.UseMarketplaceImage != nil && *c.Provider.AWS.UseMarketplaceImage)
+		(c.Provider.AWS != nil && c.Provider.AWS.UseMarketplaceImage != nil && *c.Provider.AWS.UseMarketplaceImage) ||
+		(c.Provider.OpenStack != nil && c.Provider.OpenStack.Cloud == "stackit")
```
Comment (msanft) [inline internal/config/config.go:724]: So this means that STACKIT is only accessible via marketplace images, right?

**Review by msanft — APPROVED** [review]
Thanks!

File: internal/config/config.go (hunk 2 above)
Comment (malt3) [reply internal/config/config.go:724]: Correct

File: internal/license/license.go [hunk 3]
```diff
@@ -13,6 +13,8 @@ type Action string
 const (
 	// CommunityLicense is used by everyone who has not bought an enterprise license.
 	CommunityLicense = "00000000-0000-0000-0000-000000000000"
+	// MarketplaceLicense is used by everyone who has bought an enterprise license.
```
Comment (thomasten) [inline internal/license/license.go:16]:
```suggestion
	// MarketplaceLicense is used by everyone who uses a marketplace image.
```

**Review by thomasten — APPROVED** [review]

**Pull request merged**